### PR TITLE
Feature/add number input component

### PR DESCRIPTION
**Pull request by johannchopin**
## Description

Implementation of #16.

## Checklist

- [x] I compare my branch with `develop`
- [x] All tests passed after running `yarn s-check && yarn lint`

### CODE REVIEW

File: src/components/AlphanumericInput/AlphanumericInput.svelte [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        jumpToChunk(chunkIndex + 1)
+        return
+      }
+      if (isChunkEmpty && key === KEY.backspace && !isFirstChunck) {
+        jumpToChunk(chunkIndex - 1)
+      }
+    }, 0)
+  }
+
+  const checkChunk = (i: number): void => {
```
Comment (Lyokolux) [inline src/components/AlphanumericInput/AlphanumericInput.svelte:51]: rename to `normalize_chunk` ?

File: src/components/AlphanumericInput/AlphanumericInput.svelte [hunk 2]
```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+  import { chunk } from '../../utils'
```
Comment (Lyokolux) [inline src/components/AlphanumericInput/AlphanumericInput.svelte:2]: rename to `getChunks` ?

File: src/pages/Home.svelte [hunk 3]
```diff
@@ -2,6 +2,7 @@
   import { _ } from 'svelte-i18n'
   import NextButton from '../components/NextButton.svelte'
 
+  import AlphanumericInput from '../components/AlphanumericInput/AlphanumericInput.svelte'
```
Comment (Lyokolux) [inline src/pages/Home.svelte:5]: Import line should be grouped with the other import statements

File: src/pages/Home.svelte [hunk 4]
```diff
@@ -14,3 +15,14 @@
 <NextButton>
   { $_('continue') }
 </NextButton>
+<AlphanumericInput
+  value="T890SOLA6RF6628GB8"
+  chunkLength={4}
+  onSuccess={() => alert('success')}
```
Comment (Lyokolux) [inline src/pages/Home.svelte:21]: Display something in the console instead of an alert that break the flow.

File: src/pages/Home.svelte [hunk 5]
```diff
@@ -14,3 +15,14 @@
...
+/>
+<AlphanumericInput
+  value="T890SOLA6RF6628GB8"
+  chunkLength={2}
+  onSuccess={() => alert('success')}
```
Comment (Lyokolux) [inline src/pages/Home.svelte:26]: Display something in the console instead of an alert that break the flow.

**Review by Lyokolux — COMMENTED** [review]
Great job :+1: 

It works as expected. I only have some comments to make the code more readable.

File: src/components/AlphanumericInput/AlphanumericInput.svelte [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+  import { KEY } from '../../const'
+
+  export let value: string
+  export let chunkLength: number
+  let className = ''
+  export { className as class }
```
Comment (Lyokolux) [inline src/components/AlphanumericInput/AlphanumericInput.svelte:8]: A comment to make it explicit ?

File: src/pages/Home.svelte (hunk 4 above)
Comment (johannchopin) [reply src/pages/Home.svelte:22]: This is just some 💩 code just ignore it

File: src/pages/Home.svelte (hunk 5 above)
Comment (johannchopin) [reply src/pages/Home.svelte:27]: 💩

File: src/components/AlphanumericInput/AlphanumericInput.svelte (hunk 6 above)
Comment (johannchopin) [reply src/components/AlphanumericInput/AlphanumericInput.svelte:8]: Mmmh that's just the way of doing it. Adding always a comment in each component that has a `class` prop is overkill.

**Review by Lyokolux — APPROVED** [review]

**Pull request merged**